### PR TITLE
Fix build warnings

### PR DESCRIPTION
**Pull request by tari01**
About 100000 deprecations and other build nasties...

### CODE REVIEW

**Comment by khurshid-alam** [comment]
@tari01 

When I enable tests some of the tests are failing. I am compiling on Ubuntu hirsute though.

**Comment by tari01** [comment]
> When I enable tests some of the tests are failing. I am compiling on Ubuntu hirsute though.

Yes, we still have https://github.com/AyatanaIndicators/ayatana-indicator-sound/issues/1. This is only about the warnings and the deprecated stuff.

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 project(ayatana-indicator-sound C CXX)
-cmake_minimum_required(VERSION 2.8.9)
+cmake_minimum_required(VERSION 3.16)
```
Comment (sunweaver) [inline CMakeLists.txt:2]: 3.13? Please fix as a follow-up PR.

File: src/accounts-service-access.vala [hunk 2]
```diff
@@ -130,7 +131,7 @@ public class AccountsServiceAccess : Object
         // Get master AccountsService object
         DBusProxy accounts_proxy;
         try {
-            accounts_proxy = yield DBusProxy.create_for_bus (BusType.SYSTEM, DBusProxyFlags.DO_NOT_LOAD_PROPERTIES | DBusProxyFlags.DO_NOT_CONNECT_SIGNALS, null, "org.freedesktop.Accounts", "/org/freedesktop/Accounts", "org.freedesktop.Accounts");
+            accounts_proxy = yield new DBusProxy.for_bus (BusType.SYSTEM, DBusProxyFlags.DO_NOT_LOAD_PROPERTIES | DBusProxyFlags.DO_NOT_CONNECT_SIGNALS, null, "org.freedesktop.Accounts", "/org/freedesktop/Accounts", "org.freedesktop.Accounts");
```
Comment (sunweaver) [inline src/accounts-service-access.vala:134]: this shouldn't be in this commit

File: src/accounts-service-access.vala [hunk 3]
```diff
@@ -143,9 +144,9 @@ public class AccountsServiceAccess : Object
             if (user_path_variant.check_format_string ("(o)", true)) {
                 user_path_variant.get ("(o)", out user_path);
 #if HAS_UT_ACCTSERVICE_SOUND_SETTINGS
-                _user_proxy = yield DBusProxy.create_for_bus (BusType.SYSTEM, DBusProxyFlags.GET_INVALIDATED_PROPERTIES, null, "org.freedesktop.Accounts", user_path, "com.ubuntu.AccountsService.Sound");
+                _user_proxy = yield new DBusProxy.for_bus (BusType.SYSTEM, DBusProxyFlags.GET_INVALIDATED_PROPERTIES, null, "org.freedesktop.Accounts", user_path, "com.ubuntu.AccountsService.Sound");
 #else
-                _user_proxy = yield DBusProxy.create_for_bus (BusType.SYSTEM, DBusProxyFlags.GET_INVALIDATED_PROPERTIES, null, "org.freedesktop.Accounts", user_path, "org.ayatana.AccountsService.Sound");
+                _user_proxy = yield new DBusProxy.for_bus (BusType.SYSTEM, DBusProxyFlags.GET_INVALIDATED_PROPERTIES, null, "org.freedesktop.Accounts", user_path, "org.ayatana.AccountsService.Sound");
```
Comment (sunweaver) [inline src/accounts-service-access.vala:149]: this shouldn't be in this commit

File: src/volume-warning.vala [hunk 4]
```diff
@@ -1,7 +1,6 @@
 /*
- * -*- Mode:Vala; indent-tabs-mode:t; tab-width:4; encoding:utf8 -*-
- *
```
Comment (sunweaver) [inline src/volume-warning.vala:3]: why remove these?

**Review by sunweaver — CHANGES_REQUESTED** [review]
I left some comments, please resolve.

**Review by sunweaver — APPROVED** [review]
LGTM

**Pull request merged**